lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -85,9 +85,9 @@ CHECK_EXISTING_IDENTITY:
       targetState: IPV_IDENTITY_REUSE_PAGE
     pending:
       targetState: IPV_IDENTITY_PENDING_PAGE
-    pyi-no-match:
+    fail-with-ci:
       targetState: PYI_NO_MATCH
-    fail:
```

Comment:
I don't think it should be a blocker as I suspect the answer is minimal. But is there a chance that this will bork some journeys temporarily, depending on the timings of the new versions of the process-journey-event and check-existing-identity lambdas both becoming live? I don't know how cloudformation handles that synchronisation when deploying multiple lambdas...

Reply:
This is an interesting question - I don't know how the step function deployments work. i.e. whether it is possible that one journey that a user gets the new version of the check-existing-identity lambda, but the old version of the process-journey-event lambda.

I'm inclined to leave it in this case:
- The window for such a scenario should be very small
- All these cases result in an error anyway, so it doesn't actually block any users that could complete (just degrades the messaging)

Reply:
We might need to consider some of this for the canary deployment process as well. My expectation is that we'll need to pin lambda versions in a particular step function version, if we're not already doing so.